.travis.yml

```diff
@@ -9,6 +9,8 @@ before_script:
   - rust-nightly-x86_64-unknown-linux-gnu/install.sh --prefix=$HOME/rust
   # because of this export, we can't put this into separate `install-rust.sh` script
   - export PATH=$PATH:$HOME/rust/bin
+  - cargo install rustfmt
+  - export PATH=$PATH:$HOME/.cargo/bin
```

Comment:
Hm, ideally we should not add this line, but looks like `cargo-fmt` commands needs this.